systematics/seleff/PlotCutVariationsOnePtBin.cc

```diff
@@ -100,13 +100,13 @@ void PlotCutVariationsOnePtBin(TString cfgFileName) {
                 hPromptFrac[iFile] = (TH1D*)hCrossSection[iFile]->Clone();
                 hFDFrac[iFile] = (TH1D*)hCrossSection[iFile]->Clone();
                 TGraphAsymmErrors* gFcConservative = (TGraphAsymmErrors*)infile_crossec->Get("gFcConservative");
-                for(int iPt=0; iPt<gFcConservative->GetN(); iPt++) {
+                for(int iPt=1; iPt<gFcConservative->GetN(); iPt++) {
                     double pT, promptFrac;
                     gFcConservative->GetPoint(iPt, pT, promptFrac);
```

Comment:
Now `GetPoint` start from 1 and not from 0, is it intended?

Reply:
Yes it is correct, this is due to the fact that for some reason the point `0` is set to zero in `gFcConservative`, while it is filled starting from `1`.